pkg/agent/core/ngt/handler/grpc/object_test.go

```diff
@@ -1337,6 +1337,118 @@ func Test_server_StreamGetObject(t *testing.T) {
 	}
 }
 
+func Test_server_GetTimestamp(t *testing.T) {
+	t.Parallel()
+
+	defaultConfig := config.NGT{
+		Dimension:           100,
+		DistanceType:        "l2",
+		ObjectType:          "float",
+		BulkInsertChunkSize: 10,
+		CreationEdgeSize:    20,
+		SearchEdgeSize:      10,
+		EnableProactiveGC:   false,
+		EnableCopyOnWrite:   false,
+		KVSDB: &config.KVSDB{
+			Concurrency: 10,
+		},
+		BrokenIndexHistoryLimit: 1,
+	}
+
+	setup := func(t *testing.T) (context.Context, Server) {
+		t.Helper()
+		ngt, err := service.New(&defaultConfig)
+		require.NoError(t, err)
+
+		ctx := context.Background()
+		eg, ectx := errgroup.New(ctx)
```

Comment:
I think it is better to execute eg.Wait in each test case with just in case 🤔

Reply:
[2b0715b](https://github.com/vdaas/vald/pull/2234/commits/2b0715bf2685ab24d470fb7714dc20b221339d1c)